Dockerfile-csi-controller

```diff
@@ -13,15 +13,21 @@
 # limitations under the License.
 
 FROM registry.access.redhat.com/ubi8/python-38:1-77 as builder
-COPY controller/requirements.txt /tmp/
 USER root
 RUN if [[ "$(uname -m)" != "x86"* ]]; then yum install -y rust-toolset; fi
 USER default
 RUN pip3 install --ignore-installed --default-timeout=100 --upgrade pip==21.2.4
 # avoid default boringssl lib, since it does not support z systems
 ENV GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=True
+COPY controller/requirements.txt /tmp/
 RUN pip3 install -r /tmp/requirements.txt
 
+USER root
+COPY controller/scripts/csi_general .
+RUN chmod 755 ./csi_pb2.sh
```

Comment:
is 755 the lowest value that works?
could it be submitted that way to begin with? I think it's currently with 644

Reply:
seems that it worked
done